templates/email/shropshire/archive.html

```diff
@@ -0,0 +1,55 @@
+[%
```

Comment:
We have six or so of these now, which are quite similar in places, with differences in text. Would be good to consolidate to one main shared template with sub-templates? This one is basically identical to Hounslow/Westminster, looks like.